lib/logging.js

```diff
@@ -163,8 +163,9 @@ function setLogLevel (logger, levelName) {
   const pinoLevel = PINO_LEVEL_FROM_LEVEL_NAME[levelName]
   if (!pinoLevel) {
     logger.warn('unknown log levelName "%s": cannot setLogLevel', levelName)
+  } else {
+    logger.level = pinoLevel
   }
-  logger.level = pinoLevel
```

Comment:
REVIEW NOTE: This was an existing small bug that was exposed by testing the `configLogger()` addition.